sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/DebeziumOffsetTracker.java

```diff
@@ -28,6 +29,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(DebeziumOffsetTracker.class);
 
     private DebeziumOffsetHolder restriction;
+    long toMillis = 60 * 1000;
```

Comment:
Change to constant convention
```suggestion
    private static final long MILLIS = 60 * 1000;
```